scripts/exglobal_prep_ocean_obs.py

```diff
@@ -1,17 +1,59 @@
 #!/usr/bin/env python3
 # exglobal_prep_ocean_obs.py
 # Pepares observations for marine DA
+from datetime import datetime, timedelta
+import logging
 import os
-from wxflow import YAMLFile
 import prep_marine_obs
-import logging
+import subprocess
+from wxflow import YAMLFile, save_as_yaml
+
+# set up logger
+logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+
+cyc = os.getenv('cyc')
+PDY = os.getenv('PDY')
+
+# TODO (AFE): ideally this should be an env var
```

Comment:
remove hard-coded path and define "JEDI_BIN" in the `config.obsprep` ... or whatever you called that file.